frontend/src/pages/PremiumPage/PremiumPage.jsx

```diff
@@ -0,0 +1,124 @@
+        <Registration className="header" />{" "}
+      </div>
+
+      {videos.map((video) => {
+        return <YouTube className="video" videoId={video.url} />;
+      })}
+
+      <div className="presentation">
+        <h2> Presentation</h2>
+        <p>
+          Bienvenue sur notre site <span>Origin's Digital</span> ! Nous sommes
+          ravis de vous accueillir sur notre plateforme de diffusion en ligne.
+          <br />
+          <br />
+          Notre site <span>Origin's Digital</span> vous offre un accés limité, à
+          une selection de 3 vidéos ( films, séries, documentaire ...) et est
+          accessible sur de nombreux appareils et plateformes : ordinateurs,
+          téléviseurs, smartphones, tablettes. Vous pouvez profiter de nos
+          contenus où que vous soyez.
+          <br />
+          <br />
+          Nous vous proposons également un abonnement. Vous pouvez choisir de
+          regarder vos contenus en illimité.
+          <br />
+          <br />
+          Rejoignez notre communauté et découvrez toutes les nouveautés et les
+          avantages que nous vous réservons. Nous espérons que vous apprécierez
+          votre expérience sur notre site <span>Origin's Digital</span> !
+        </p>
+      </div>
+      <div className="chart">
+        <img className="chart" src={chart} alt="chart" />
+      </div>
+      <div className="goPremium">
+        <button
+          value="Go Premium"
+          className="goPremium1"
+          onClick={togglePremiumSelected}
+        >
+          DEVENIR PREMIUM
+        </button>
+      </div>
+      <div className="Ok">
+        <Modal
+          isShowing={isPremiumSelectedShowed}
+          hide={togglePremiumSelected}
+          title="Registration field :"
+        >
+          <form>
+            <div className="formGroup">
+              <input type="text" placeholder="Firstname" />
+            </div>
+            <div className="formGroup">
+              <input type="text" placeholder="Lastname" />
+            </div>
+            <div className="formGroup">
+              <input type="text" placeholder="Adress" />
+            </div>
+            <div className="formGroup">
+              <input type="text" placeholder="City" />
+            </div>
+            <div className="formGroup">
+              <input type="text" placeholder="Credit Card Numbers" />
+            </div>
+            <div className="formGroup">
+              <input type="text" placeholder="CVV" />
+            </div>
+            <div className="formGroup">
+              <input
+                type="text"
+                placeholder="Expiration Date (Day/Month/Year)"
+              />
+            </div>
+            <div className="formGroup">
+              <input
+                type="submit"
+                value="Confirm"
+                onClick={(evt) => {
+                  evt.preventDefault();
+                  navigate("/home");
+                }}
```

Comment:
It's not a good idea to use a confirm button to redirect (a Link'd be more adapted)
But since that's for the payment funnel and you won't implement this, I'll let it pass